spec/appcenter_upload_spec.rb

```diff
@@ -1373,6 +1380,8 @@ def stub_add_to_destination(status, app_name = "app", owner_name = "owner", dest
       stub_get_destination(200)
       stub_add_to_destination(200)
       stub_get_release(200)
+      stub_fetch_distribution_groups(owner_name: 'owner', app_name: 'app')
```

Comment:
Can you please add another test case where app should have the group and skips adding it.
So that it covers this code lines
`  
  destination_names.each do |destination_name|
`
`
           unless group_names.include? destination_name
`

Reply:
added!

Made sure both scenarios are checked with 2 destinations so it sends the request for one but not the other 👍